### PR TITLE
add docs for multi-modal examples

### PR DESCRIPTION
**Pull request by natolambert**
closes #54, should be merged with #105.

### CODE REVIEW

**Review by patrickvonplaten — APPROVED** [review]
Ok for me! @pcuenca could you also take a look here? :-)

File: docs/source/using-diffusers/other-modalities.mdx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Using Diffusers with other modalities
+
+Diffusers is in the process of expanding to modalities other than images.
+
+Currently, the only example is for [molecule conformation](https://www.nature.com/subjects/molecular-conformation#:~:text=Definition,to%20changes%20in%20their%20environment.) generation.
```
Comment (pcuenca) [inline docs/source/using-diffusers/other-modalities.mdx:17]:
```suggestion
Currently, one example is for [molecule conformation](https://www.nature.com/subjects/molecular-conformation#:~:text=Definition,to%20changes%20in%20their%20environment.) generation.
```

We also have audio, don't we? Maybe cross-link there too?

**Review by pcuenca — APPROVED** [review]
Awesome, thanks!

**Comment by pcuenca** [comment]
Sorry for the delay @natolambert and @patrickvonplaten, I missed the notification somehow :(

File: docs/source/using-diffusers/other-modalities.mdx (hunk 1 above)
Comment (natolambert) [reply docs/source/using-diffusers/other-modalities.mdx:17]: I made a entire page for audio as I think it'll grow (much like I made a page for RL)

**Pull request merged**